cra-rxjs-styled-components/src/components/issue/IssueTabHeader/IssueTabHeader.tsx

```diff
@@ -0,0 +1,77 @@
+import { useCallback, useState } from 'react';
+import DetailsDropdown from '../../DetailsDropdown';
+import CorrectIcon from '../../icons/CorrectIcon';
+import IssueIcon from '../../icons/IssueIcon';
+import type { IssueTabValues } from '../types';
+
+import { Container, StatusLabel, StatusTab } from './IssueTabHeader.style';
+import type { DropdownTitle } from '../types';
+
+interface Props {
+  toggleTab: any;
+}
+
+export default function IssueTabHeader(props: Props) {
```

Comment:
Total nitpick on this one. If there's only one prop can we destructure within the function signature instead of within the component?